### DISCUSSION TITLE
Do post request to external api

### DISCUSSION THREAD

**Discussion by VishnuVardhan-Dwarampudi** [loopbackio/loopback-next#7896] [Q&A]
I am trying to do post request to external api with json body, from loopback4 application. Please share some examples.

**Comment by achrinza** [comment]
Would it be possible to use an existing HTTP library (e.g. Axios, Got)? A LoopBack 4 project is still a Node.js project at its core, so the usual Node.js knowledge and tricks still apply.

**Comment by mrmodise** [comment]
Am going to update this section to demonstrate how to do this: https://loopback.io/doc/en/lb4/Calling-rest-apis.html#add-the-rest-endpoints
